Palace.xcodeproj/project.pbxproj

```diff
@@ -2491,7 +2480,7 @@
 		A823D804192BABA400B55DE2 = {
 			isa = PBXGroup;
 			children = (
-				E585662026979AC100A5FBD5 /* NYPLAEToolkit.xcodeproj */,
+				E585662026979AC100A5FBD5 /* ios-drm-audioengine */,
```

Comment:
What is the benefit of removing NYPLAEToolkit project and replacing it with the folder? From what I see, Xcode won't build it and any change in the toolkit won't take effect until Carthage rebuilds it.